presto-tpcds/src/test/java/com/facebook/presto/tpcds/TestTpcdsRecordSet.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tpcds;
+
+import com.facebook.presto.spi.RecordCursor;
+import com.facebook.presto.spi.RecordSet;
+import com.google.common.collect.ImmutableList;
+import com.teradata.tpcds.Results;
+import com.teradata.tpcds.Session;
+import com.teradata.tpcds.Table;
+import com.teradata.tpcds.column.Column;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.facebook.presto.common.type.BigintType.BIGINT;
+import static com.facebook.presto.common.type.CharType.createCharType;
+import static com.facebook.presto.common.type.DateType.DATE;
+import static com.facebook.presto.common.type.DecimalType.createDecimalType;
+import static com.facebook.presto.common.type.IntegerType.INTEGER;
+import static com.facebook.presto.common.type.TimeType.TIME;
+import static com.facebook.presto.common.type.VarcharType.createVarcharType;
+import static com.teradata.tpcds.Table.CALL_CENTER;
+import static com.teradata.tpcds.Table.DBGEN_VERSION;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_CALL_CENTER_SK;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_MANAGER;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_NAME;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_REC_END_DATE;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_REC_START_DATE;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_SQ_FT;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_STATE;
+import static com.teradata.tpcds.column.CallCenterColumn.CC_TAX_PERCENTAGE;
+import static com.teradata.tpcds.column.DbgenVersionColumn.DV_CREATE_DATE;
+import static com.teradata.tpcds.column.DbgenVersionColumn.DV_CREATE_TIME;
+import static com.teradata.tpcds.column.DbgenVersionColumn.DV_VERSION;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestTpcdsRecordSet
+{
+    private final Session session = Session.getDefaultSession().withScale(.01);
+
+    @Test
+    public void testGetColumnTypes()
+    {
+        Table table = Table.getTable(CALL_CENTER.getName());
```

Comment:
sorry, missed these too.  We use static import for basically everything unless there would be some ambiguity without the prefix.  so for here: static import Table.getTable and Results.constructResults

Reply:
no worries, thanks for catching these!